vmdb/spec/models/container_node_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "spec_helper"
+
+describe ContainerNode do
+  before do
+    @container_node = FactoryGirl.create(:container_node)
+  end
+
+  it "can be associated to a Vmware VM" do
+    associate_container_with_vm(:vm_vmware, @container_node)
+  end
+
+  it "can be associated to an Openstack instance" do
+    associate_container_with_vm(:vm_openstack, @container_node)
+  end
+
+  def associate_container_with_vm(vm_factory_name, container_node)
+    vm = FactoryGirl.create(vm_factory_name)
+    container_node.lives_on = vm
+    container_node.save
+
+    container_node.lives_on.should == vm
+  end
```

Comment:
I feel like these specs are just testing AR, so aren't really needed?  Maybe if you want to ensure polymorphic, then point to a VM and then point to something else like a ConfiguredSystem ?